src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -12,30 +13,50 @@
 import java.util.List;
 
 public class JobRoleDao {
-    public List<JobRole> getJobRoles() throws SQLException {
-        List<JobRole> jobRoles = new ArrayList<>();
+
+    public List<JobRoleResponse> getJobRoles(
```

Comment:
Rename this method to getAllOpenJobRoles()

Reply:
Resolved